tests/functional/spec/list_collector_section_summary_items.spec.js

```diff
@@ -0,0 +1,188 @@
+      addCompany("Company A", "123", true);
+      anyMoreCompaniesNo();
+      answerUkBasedQuestion();
+      expect($(SectionSummaryPage.companiesListLabel(1)).getText()).to.contain("Name of UK company or branch");
+      expect($(companiesListRowItem(1, 1)).getText()).to.contain("Company A");
+      expect($(companiesListRowItem(1, 2)).getText()).to.contain("123");
+      expect($(companiesListRowItem(1, 3)).getText()).to.contain("Yes");
+      expect($(companiesListRowItemAnchor(1)).getHTML()).to.contain("return_to=section-summary#company-or-branch-name");
+      expect($(companiesListRowItemAnchor(2)).getHTML()).to.contain("return_to_answer_id=registration-number#registration-number");
+      expect($(companiesListRowItemAnchor(3)).getHTML()).to.contain("return_to_answer_id=authorised-insurer-radio#authorised-insurer-radio");
+    });
+    it("When I add multiple items, Then all the items should be visible on the section summary and have correct values", () => {
+      drivingQuestionYes();
+      addCompany("Company A", "123", true);
+      anyMoreCompaniesYes();
+      addCompany("Company B", "456", false);
+      anyMoreCompaniesYes();
+      addCompany("Company C", "789", true);
+      anyMoreCompaniesNo();
+      answerUkBasedQuestion();
+      expect($(companiesListRowItem(1, 1)).getText()).to.contain("Company A");
+      expect($(companiesListRowItem(1, 2)).getText()).to.contain("123");
+      expect($(companiesListRowItem(1, 3)).getText()).to.contain("Yes");
+      expect($(companiesListRowItem(2, 1)).getText()).to.contain("Company B");
+      expect($(companiesListRowItem(2, 2)).getText()).to.contain("456");
+      expect($(companiesListRowItem(2, 3)).getText()).to.contain("No");
+      expect($(companiesListRowItem(3, 1)).getText()).to.contain("Company C");
+      expect($(companiesListRowItem(3, 2)).getText()).to.contain("789");
+      expect($(companiesListRowItem(3, 3)).getText()).to.contain("Yes");
+    });
+    it("When I remove an item, Then the list of answers should no longer be visible on the section summary.", () => {
+      drivingQuestionYes();
+      addCompany("Company A", "123", true);
+      anyMoreCompaniesNo();
+      answerUkBasedQuestion();
+      removeFirstCompany();
+      expect(browser.getUrl()).to.contain(SectionSummaryPage.url());
+      expect($(SectionSummaryPage.companiesListEditLink(1)).isExisting()).to.be.false;
+      expect($(SectionSummaryPage.companiesListRemoveLink(1)).isExisting()).to.be.false;
+    });
+    it("When I remove an item but the list collector is still on the path, Then the placeholder text should be visible on the section summary.", () => {
+      drivingQuestionYes();
+      addCompany("Company A", "123", true);
+      anyMoreCompaniesNo();
+      answerUkBasedQuestion();
+      removeFirstCompany();
+      expect(browser.getUrl()).to.contain(SectionSummaryPage.url());
+      expect($("body").getText()).to.contain("No UK company or branch added");
+    });
```

Comment:
Should have another remove test where you have more than 1 item, to show only the single item you removed is gone.

Reply:
Added additional test 👍